modules/packages/Socket.chpl

```diff
@@ -408,13 +665,13 @@ record tcpListener {
     const client = server.accept()
 
   :arg timeout: time to wait for new connection.
-  :type timeval: :type:`~Sys.timeval`
+  :type timeout: `struct_timeval`
```

Comment:
Is it possible to keep this a working cross-reference using something like :type:`~OS.POSIX.struct_timeval`?  (where the question in my mind is whether we have working submodule qualification support).

Reply:
The documentation CI test was failing with `:type timeout: `~OS.POSIX.struct_timeval`. 

I'll try omitting the `OS.` and a few other things to see if I can get it to work.

Reply:
(oh, though later, it looks like maybe you were able to make this work using simply `~POSIX`?